### PR TITLE
ACS-4818 Tests for wildcard queries.

### PR DESCRIPTION
**Pull request by tpage-alfresco**
We no longer escape asterisks in wildcard queries - add tests for this.

### CODE REVIEW

**Review by gevagel — APPROVED** [review]
LGTM

**Review by KikoUK — APPROVED** [review]

**Review by krdabrowski — APPROVED** [review]

**Review by azakrzewski-hy — APPROVED** [review]

File: tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchLiveIndexingTests.java [hunk 1]
```diff
@@ -102,7 +103,7 @@ public void dataPreparation()
         dataUser.addUserToSite(userMultiSite, siteModel1, UserRole.SiteContributor);
         dataUser.addUserToSite(userMultiSite, siteModel2, UserRole.SiteContributor);
 
-        createContent(FILE_0_NAME, "This is the first test containing the unique word " + UNIQUE_WORD, siteModel1, userSite1);
+        file0 = createContent(FILE_0_NAME, "This is the first test containing the unique word " + UNIQUE_WORD, siteModel1, userSite1);
```
Comment (mpichura) [inline tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchLiveIndexingTests.java:106]: Is this variable ever used in the test class?

**Review by mpichura — APPROVED** [review]
LGTM - had one question - but it's  a minor one.
Don't forget about removing snapshots.

File: tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchLiveIndexingTests.java (hunk 1 above)
Comment (tpage-alfresco) [reply tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchLiveIndexingTests.java:106]: It's not any more - thanks!

File: pom.xml [hunk 2]
```diff
@@ -10,12 +10,12 @@
         <groupId>org.alfresco</groupId>
         <artifactId>alfresco-enterprise-repo</artifactId>
         <relativePath>../alfresco-enterprise-repo/pom.xml</relativePath>
-        <version>20.99</version>
+        <version>20.100-SNAPSHOT</version>
```
[the remark this answers is not in the document]
Comment (tpage-alfresco) [reply pom.xml]: Yes - this was just needed to run the tests while the production code was on a feature branch.

**Comment by tpage-alfresco** [comment]
Failures are also happening on master. I'll create a ticket to investigate this.

**Comment by tpage-alfresco** [comment]
Failures are also happening on master. I'll create a ticket to investigate this.

**Pull request merged**